applications/ParticleMechanicsApplication/custom_conditions/grid_based_conditions/mpm_grid_base_load_condition.cpp

```diff
@@ -29,33 +29,33 @@ namespace Kratos
     {
         KRATOS_TRY
 
-        GeometryType& rGeom = GetGeometry();
-        const unsigned int NumberOfNodes = rGeom.size();
-        const unsigned int dim = rGeom.WorkingSpaceDimension();
-        if (rResult.size() != dim * NumberOfNodes)
+        GeometryType& r_geometry = GetGeometry();
```

Comment:
if you would make a separate PR for the renamings the review would be much easier
Now I am really lost

Reply:
@philbucher it used to be okay, I am just following @tteschemacher comments